README.md

```diff
@@ -92,9 +92,16 @@ go get -u github.com/golang/protobuf/protoc-gen-go
 go get -u github.com/jteeuwen/go-bindata/...
 ```
 
-You need to get and build [v8worker2](https://github.com/ry/v8worker2).  The package will not build with `go
-get` and will log out an error, which can be ignored. It takes about 30 minutes
-to build:
+You need to get and build [v8worker2](https://github.com/ry/v8worker2).  __The package will not build with `go
+get` and will log out an error ⚠__
+```bash
+# pkg-config --cflags v8.pc
+Failed to open 'v8.pc': No such file or directory
+No package 'v8.pc' found
+pkg-config: exit status 1
+```
+
+, __which can be ignored__. It takes about 30 minutes to build:
```

Comment:
BTW, i think the ","(comma symbol) are not needed as well.

Reply:
Sounds good to me, thanks for taking such a close look.